<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="thermalFeed">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>311</width>
      <height>261</height>
     </rect>
    </property>
    <property name="text">
     <string>Seek thermal camera not found!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="heatmapChoice">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>280</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Grey</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hot</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Autumn</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bone</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Jet</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Winter</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rainbow</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ocean</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Summer</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Spring</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cool</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HSV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pink</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>290</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Colourmap: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Thermal Camera Feed</string>
    </property>
   </widget>
   <widget class="QLabel" name="colourbarFeed">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>51</width>
      <height>261</height>
     </rect>
    </property>
    <property name="text">
     <string>Colour Bar</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>340</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Temperature Unit:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="temperatureChoice">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>330</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Celsius</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Kelvin</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fahrenheit</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="highT">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>20</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lowT">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>240</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="emmisivityChoice">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>280</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Skin (0.98)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Matte (0.97)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Semi-matte (0.80)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Semi-glossy (0.60)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Glossy (0.30)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Black body (1.0)</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>280</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Emmisivity Correction:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>380</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Hightlight:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="highlight">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>380</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Hottest Point</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Coldest Point</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Face Temperature:</string>
    </property>
   </widget>
   <widget class="QLabel" name="faceTemp">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>No Faces Detected!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>330</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Circle Faces:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>330</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>430</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Alarm</string>
    </property>
   </widget>
   <widget class="QLabel" name="alarm">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>100</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/alarm.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
